packages/Webkul/Theme/src/Repositories/ThemeCustomizationRepository.php

```diff
@@ -19,24 +19,57 @@ public function model(): string
         return ThemeCustomization::class;
     }
 
+    /**
+     * Update the specified theme
+     *
+     * @param  array  $data
+     * @param  int  $id
+     */
+    public function update($data, $id): ThemeCustomization
+    {
+        $locale = core()->getRequestedLocaleCode();
+
+        if ($data['type'] == 'static_content') {
+            $data[$locale]['options']['html'] = preg_replace('/<script\b[^>]*>(.*?)<\/script>/is', '', $data[$locale]['options']['html']);
+            $data[$locale]['options']['css'] = preg_replace('/<script\b[^>]*>(.*?)<\/script>/is', '', $data[$locale]['options']['css']);
+        }
+
+        $data['status'] = request()->input('status') == 'on';
```

Comment:
Request class cannot be used inside the repository.

Reply:
done